libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -695,6 +709,69 @@ bool AP_AHRS_NavEKF::get_relative_position_NED(Vector3f &vec) const
     }
 }
 
+// return a relative ground position in meters/second, North/East
```

Comment:
I suppose it's ok to mention that it "returns" both, it's pretty clear that it actually returns a bool and permutes the reference var passed in